triton-vm/src/stark.rs

```diff
@@ -1770,6 +1763,58 @@ pub(crate) mod triton_stark_tests {
         assert!(result.unwrap());
     }
 
+    #[test]
+    #[ignore = "used for catching deserialization errors"]
```

Comment:
Why ignore the tests?

Reply:
These tests do not catch errors that `triton_prove_verify_halt` does not already catch. They are useful only for tracking them down. I guess I should replace the comment "catching" with "tracking".